### ISSUE TITLE
ESPHome, СВТ-15

### ISSUE THREAD

**Issue by kirkorblack** [vooon/elehant-to-mqtt#11]
А будет ли доработка для ESPHome для двухтарифных счетчиков?
Есть 4 счетчика, 3 однотарифных, и один двух тарифный

1. Серийный номер на двухтарифном счетчике 0004706 (как прописывать? 5 цифр или 4)
2. С двухтарифного не собирает вообще никакой инфы (В приложении отображается, температура, счетчик холодной воды пока не нагрелась и счетчик горячей воды, после того как нагрелась до 40 градусов)

Очень хотелось бы чтобы оно именно на ESPHome заработало...

**Comment by vooon** [comment]
Esphome версию я забросил, она меньше сообщений со счетчиков ловит, прежде чем упасть. Я до сих пор не знаю что там не правильно с бле стеком...

Поддержку СВТ-15 я портировал из https://github.com/raxers/elehant_water в основную прошивку, но не тестировал, т.к. у меня их нет.

Обычно ведуще нули отбрасываются.

**Comment by fairslaven** [comment]
Подскажите, Не получается добавить двух тарифный счетчик Элехант в НА. Сенсоры видны но в них данные не поступают. скажите в чем может быть проблема?

**Comment by fairslaven** [comment]
В 4 метрах  и одна бетонная стена. Как посмотреть что поступает в MQTT?

**Comment by fairslaven** [comment]
Bluetooth настроен через ESPhome с помощью Bluetooth Proxy

**Comment by vooon** [comment]
ESPHome версию я забросил, на момент когда написал чаще теряла пакеты.

4м через бетон может быть много, передатчик там слабенький. Попробуйте телефоном получить данные из места установки еспшки.

**Comment by fairslaven** [comment]
Данные на телефон я получаю. А как тогда иначе, если не через ESPHome подключить счетчик?

**Comment by vooon** [comment]
Так вот в корне же лежит platformio проект.

**Comment by fairslaven** [comment]
Этот код надо загрузить в ESP? Или что сделать? подскажите пожалуйста

**Comment by vooon** [comment]
Да. Ставите platformio, потом подключаете плату по юсб и запускаете `pio run -t upload`.

**Comment by vooon** [comment]
А, даже и ставить его не нужно, ESPHome использует PIO, так что где esphome run делаете, там и pio run должен быть.

**Comment by fairslaven** [comment]
Можно чуть чуть по подробней. Пока что ничего не понятно.
